core/templates/pages/android-page/android-page.component.html

```diff
@@ -0,0 +1,185 @@
+<div class="oppia-android-page-container e2e-test-about-foundation-page">
```

Comment:
The classname should be something like `e2e-test-android-page` or similar. Remove this class if it doesn't affect e2e tests.

Reply:
Removed it